conf/supervisord-dev.conf

```diff
@@ -0,0 +1,50 @@
+[supervisord]
```

Comment:
You can directly compare the commands in this file to the ones used in prod: https://github.com/hypothesis/h/blob/master/conf/supervisord.conf